docs/products/kafka/howto/kafka-sasl-auth.md

```diff
@@ -1,28 +1,184 @@
-    project and choose your Aiven for Apache Kafka® service.
-2.  In the service page, select **Service settings** from the sidebar.
-3.  On the **Service settings** page, scroll down to the **Advanced
-    configuration** section.
-4.  Click **Configure**.
-5.  In the **Advanced configuration** dialog, set the
-    `kafka_authentication_methods.sasl` toggle to the enabled position.
-6.  Click **Save configuration**.
+## Enable SASL authentication
 
-The **Connection information** at the top of the **Overview** page will
-now offer the ability to connect via SASL or via Client Certificate.
+<Tabs groupId="config-methods">
+<TabItem value="console" label="Aiven Console" default>
 
-![Choose between SASL and certificate connection details](/images/content/products/kafka/sasl-connect.png)
+1. Access the [Aiven Console](https://console.aiven.io) and select your
+   Aiven for Apache Kafka service.
+1. Click <ConsoleLabel name="Service settings"/>.
+1. Scroll to **Advanced configuration** and click **Configure**.
+1. In the **Advanced configuration** window, set `kafka_authentication_methods.sasl` to
+   **Enabled**.
+1. Click **Save configurations**.
+
+The **Connection information** in the **Overview** page now allows connections via SASL
+or Client certificate.
 
 :::note
 Although these connections use a different port, the host, CA, and user
 credentials remain consistent.
 :::
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+Enable SASL authentication for your Aiven for Apache Kafka service using
+[Aiven CLI](/docs/tools/cli):
+
+1. Get the name of the Aiven for Apache Kafka service:
+
+   ```bash
+   avn service list
+   ```
+
+   Note of the `SERVICE_NAME` corresponding to your Aiven for Apache Kafka service.
+
+1. Enable SASL authentication:
+
+   ```bash
+   avn service update <SERVICE_NAME> -c kafka_authentication_methods.sasl=true
+   ```
+
+   Parameters:
+
+   - `<SERVICE_NAME>`: Name of your Aiven for Apache Kafka service.
+   - `kafka_authentication_methods.sasl`: Set to `true` to enable SASL authentication.
+
+</TabItem>
+<TabItem value="api" label="API">
+
+Use the [ServiceUpdate](https://api.aiven.io/doc/#tag/Service/operation/ServiceUpdate)
+API to enable SASL authentication on an existing service:
+
+  ```bash
+   curl -X PUT "https://console.aiven.io/v1/project/{project_name}/service/{service_name}" \
+     -H "Authorization: Bearer <API_TOKEN>" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "user_config": {
+             "kafka_authentication_methods": {
+               "sasl": true
+             }
+          }
+       }'
+   ```
+
+   Parameters:
+
+   - `project_name`: Name of your Aiven project.
+   - `service_name`: Name of your Aiven for Apache Kafka service.
+   - `API_TOKEN`: API token for [authentication](/docs/platform/howto/create_authentication_token).
+   - `kafka_authentication_methods.sasl`: Set to `true` to enable SASL authentication.
+
+</TabItem>
+</Tabs>
+
+## Configure SASL mechanisms
```

Comment:
Is it clear here that `kafka_sasl_mechanisms.plain` and the other two are by default enabled, i.e. if *not* configured, they're enabled. Or to put it the other way around, only if explicitly disabled they are disabled.

Reply:
I updated the content to clarify this point further.